{{cookiecutter.project_slug}}/installer.py

```diff
@@ -0,0 +1,70 @@
+#!/bin/python
+
+import subprocess as sp
+from argparse import ArgumentParser, BooleanOptionalAction
+
+
+def condacmd(cmd : str):
+    '''Wrapper for conda shell commands.
+    
+    '''
+    proc =  sp.run(f"conda {cmd}", shell=True, stdout=sp.PIPE, stderr=sp.STDOUT)
```

Comment:
I would normally run with check=True, to ensure that in case the command fails, python will raise an exception (unless there is a reason to want a behaviour where python continues when the command fails.